src/vstart.sh

```diff
@@ -671,6 +671,20 @@ EOF
         run 'mgr' $CEPH_BIN/ceph-mgr -i $name $ARGS
     done
 
+    wait_active_start=`date +%s`
+    while [ 1 ]; do
+      if ceph_adm -s 2>&1 | grep 'mgr:.*\?(active)' >&/dev/null; then
+        break
+      fi
+      echo "Waiting for MGR to become active."
+      waited_for=$((`date +%s` - wait_active_start))
```

Comment:
I really do not like this construct.
In other scripts the sequence is something like
` for w in \`seq 1 $timeout}\`; do .....; if oke; then break; done`
And as such obvious need to work with `date +%s` and math 

Also printing "Waiting ...." every sec is going to uselessly polute the output log